src/boot/bootctl.c

```diff
@@ -123,12 +123,8 @@ static int verify_esp(
 
         errno = 0;
         b = blkid_new_probe_from_filename(t);
-        if (!b) {
-                if (errno == 0)
-                        return log_oom();
-
-                return log_error_errno(errno, "Failed to open file system \"%s\": %m", p);
-        }
+        if (!b)
+                return log_error_errno(errno ?: ENOMEM, "Failed to open file system \"%s\": %m", p);
```

Comment:
yupp, but maybe we can find a better name? log_error_make() or so? or log_error_set()? or log_error_issue() or so?

also, I figure you mean `errno != 0`, not `errno > 0`... Also, don't use `errno` at all here, since it's a weirdly defined macro...